src/System.Windows.Forms/src/System/Windows/Forms/ErrorProvider.cs

```diff
@@ -192,6 +192,17 @@ public ContainerControl ContainerControl
             }
         }
 
+        /// <summary>
+        /// Determine if the controls that are linked to this ErrorProvider currently have errors.
```

Comment:
```suggestion
        ///  Determines if the this <see cref="ErrorProvider"/> has any errors for any of the associated controls.
```

@gewarren does this look ok to you?

Reply:
Technically it should look like this:

```suggestion
        /// Gets a value that indicates if this <see cref="ErrorProvider"/> has any errors for any of the associated controls.
```